examples/c#/AnnotateDocumentExample/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Grpc.Core;
+using Grpc.Net.Client;
+using Ssn.Annotator.V1;
+using Confidence = Ssn.Type.Confidence;
+using Document = Ssn.Annotator.V1.Document;
+
+namespace ConsoleApp1;
+
+static class Program {
+    public static void Main(string[] args) {
+        annotateDocument();
+    }
+
+    static void annotateDocument()
+    {
+        //channel
+        var chnl = GrpcChannel.ForAddress("https://api.snbx.ssn.visma.ai:443");
```

Comment:
I'd suggest calling the staging env here - it will also make people aware that there is such an environment. So, in this case it would be `https://api.stag.ssn.visma.ai:443`

Reply:
And on all the subsequent calls as well

Reply:
Is it okay also for development purposes? Right now I added todo to change it to stag in the end. Can it be changed right now even when we might still be sending requests?

Reply:
Yes it's totally fine to change it to staging - it's kind of like our live testing environment, plus you can't really take it down with a few annotator requests anyway 😄 So go ahead and change it to staging, it's completely fine